articles/draas/draas-sco.md

```diff
@@ -113,23 +113,23 @@ You can self-manage your protected environment via the Zerto Self-Service Portal
 
 - Perform live failovers
 
-We've disabled some areas of the Zerto Self-Service Portal as they're for services we don't presently support. These include:
+We've disabled some areas of the ZSSP as they're for services we don't presently support. These include:
 
-- Backup to external systems. At present, you cannot export your journal to external backup targets (such as UKCloud Cloud Storage). We've identified a compliance issue with this solution which we're working to address. Until this is addressed, the feature will remain unavailable.
+- Backup to external systems. At present, you cannot export your journal to external backup targets (such as UKCloud Cloud Storage). We've identified a compliance issue with this solution, which we're working to address. Until this is addressed, the feature will remain unavailable.
 
 Additionally, the following scenarios cannot be protected:
 
 - VMs with virtual disks with individual disks greater than 3TB
 
-- VMs which are clustered using shared virtual disks
+- VMs that are clustered using shared virtual disks
 
 ## Alerting and reporting
 
-The Zerto Self-Service Portal provides live real-time reporting of your solution and the current level of protection being provided to your VMs.
+The Zerto Self-Service Portal (ZSSP) provides live real-time reporting of your solution and the current level of protection being provided to your VMs.
```

Comment:
ZSSP

Reply:
Remove "The Zerto Self-Service Portal (ZSSP)".

The ZSSP provides live real-time reporting......